### PR TITLE
[monorepo] updated releaser with changes in the new version

### PR DESCRIPTION
**Pull request by grossmannmartin**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Shopsys Platform made some changes that needed to be addressed in the release process.
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

----
:globe_with_meridians: Live Preview:
  - https://mg-fix-releaser.odin.shopsys.cloud
  - https://cz.mg-fix-releaser.odin.shopsys.cloud

### CODE REVIEW

File: utils/releaser/src/ReleaseWorker/ReleaseCandidate/CheckLatestVersionOfReleaserReleaseWorker.php [hunk 1]
```diff
@@ -39,8 +39,8 @@ public function work(
         string $initialBranchName = AbstractShopsysReleaseWorker::MAIN_BRANCH_NAME,
     ): void {
         $this->symfonyStyle->note(
-            'It is possible, that in current master branch there is improved version of Releaser.
-            Check that and update releaser in version branch by replacing Releaser directory with one from master branch.',
+            'It is possible, that in current default branch there is improved version of Releaser.
```
Comment (TomasLudvik) [inline utils/releaser/src/ReleaseWorker/ReleaseCandidate/CheckLatestVersionOfReleaserReleaseWorker.php]: What about default (highest numbered)

File: utils/releaser/src/ReleaseWorker/ReleaseCandidate/BuildProjectBaseOnHeimdallReleaseWorker.php [hunk 2]
```diff
@@ -1,45 +0,0 @@
...
-
-declare(strict_types=1);
-
-namespace Shopsys\Releaser\ReleaseWorker\ReleaseCandidate;
-
-use PharIo\Version\Version;
-use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
-use Shopsys\Releaser\Stage;
-
-final class BuildProjectBaseOnHeimdallReleaseWorker extends AbstractShopsysReleaseWorker
```
Comment (TomasLudvik) [inline utils/releaser/src/ReleaseWorker/ReleaseCandidate/BuildProjectBaseOnHeimdallReleaseWorker.php:11]: I would keep this and ask a developer to check the GitHub Actions status of the project-base.

File: utils/releaser/src/ReleaseWorker/ReleaseCandidate/SendBranchForReviewAndTestsReleaseWorker.php [hunk 3]
```diff
@@ -30,9 +30,13 @@ public function work(
...
     ): void {
-        $this->symfonyStyle->note(
-            'Keep in mind that if there are any notes from the code review or tests, you need to address them, commit and push the fixes, force-split your branch using tool-monorepo-force-split-branch and build test-rc-project-base special job on Heimdall',
-        );
+        $this->symfonyStyle->note(['Keep in mind that if there are any notes from the code review or tests, you need to:',
+            ' - address them',
+            ' - commit and push the fixes',
+            ' - force-split your branch using GitHub Actions',
+            '       https://github.com/shopsys/shopsys/actions/workflows/monorepo-force-split-branch.yaml',
+            ' - run the tests again',
```
Comment (TomasLudvik) [inline utils/releaser/src/ReleaseWorker/ReleaseCandidate/SendBranchForReviewAndTestsReleaseWorker.php:38]: add check of project-base here too

**Review by TomasLudvik — CHANGES_REQUESTED** [review]

File: utils/releaser/src/ReleaseWorker/ReleaseCandidate/CheckLatestVersionOfReleaserReleaseWorker.php (hunk 1 above)
Comment (grossmannmartin) [reply utils/releaser/src/ReleaseWorker/ReleaseCandidate/CheckLatestVersionOfReleaserReleaseWorker.php]: changed

File: utils/releaser/src/ReleaseWorker/ReleaseCandidate/BuildProjectBaseOnHeimdallReleaseWorker.php (hunk 2 above)
Comment (grossmannmartin) [reply utils/releaser/src/ReleaseWorker/ReleaseCandidate/BuildProjectBaseOnHeimdallReleaseWorker.php:11]: good idea

File: utils/releaser/src/ReleaseWorker/ReleaseCandidate/SendBranchForReviewAndTestsReleaseWorker.php (hunk 3 above)
Comment (grossmannmartin) [reply utils/releaser/src/ReleaseWorker/ReleaseCandidate/SendBranchForReviewAndTestsReleaseWorker.php:38]: added

**Review by TomasLudvik — APPROVED** [review]

**Review by pk16011990 — APPROVED** [review]

**Pull request merged**